### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Gustavo-1300**
Pull request with empty history

### CODE REVIEW

File: life_expectancy/tests/test_cleaning.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+def test_save_data(fixture_expect):
+    """Run the 'save_data'"""
+    with patch.object(fixture_expect, 'to_csv') as to_csv_mock:
+        to_csv_mock.side_effect = print('Data saved', end="")
+        save_data(
+            fixture_expect,
+            'pt_life_expectancy.csv',
+            OUTPUT_DIR
+        )
+        to_csv_mock.assert_called_once()
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_cleaning.py]:
```suggestion
def test_save_data(fixture_expect, capfd):
    """Run the 'save_data'"""
    with patch.object(fixture_expect, 'to_csv') as to_csv_mock:
        to_csv_mock.side_effect = print('Data saved', end="")
        save_data(
            fixture_expect,
            'pt_life_expectancy.csv',
            OUTPUT_DIR
        )
        to_csv_mock.assert_called_once()
        # capfd is a Pytest fixture that captures the output of the print statement
        out, _ = capfd.readouterr()
        assert out == "Data saved"
```

File: life_expectancy/tests/test_cleaning.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@pytest.fixture
+def fixture_expect():
+    """
+    Load mock expected data
+    :return: Dict with prepared data mocked
+    """
+
+    return pd.DataFrame(data_expect())
+
+@patch("pandas.read_table")
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_cleaning.py]:
```suggestion
@patch("life_expectancy.cleaning.pd.read_table")
```

File: life_expectancy/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+@pytest.fixture(autouse=True)
+def run_before_and_after_tests() -> None:
+    """Fixture to execute commands before and after a test is run"""
+    # Setup: fill with any logic you want
+
+    yield # this is where the testing happens
+
+    # Teardown : fill with any logic you want
+    file_path = OUTPUT_DIR / "pt_life_expectancy.csv"
+    file_path.unlink(missing_ok=True)
```
Comment (MrCordeiro) [inline life_expectancy/tests/conftest.py]: **suggestion**: This bit of code was meant to clean the extra file that the test would create inside the OUTPUT_DIR. But now you have correctly made your tests consume fixture, I think you can safely remove this.

File: life_expectancy/tests/test_cleaning.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+def test_load_data(read_table_mock: Mock, fixture_raw):
```
Comment (MrCordeiro) [inline life_expectancy/tests/test_cleaning.py:28]: I loved that you used the Mock type hint! :D

**Review by MrCordeiro — APPROVED** [review]
I made a few comments, but none of them are blocking!
Nice work, man!

File: life_expectancy/tests/test_cleaning.py (hunk 2 above)
Comment (Gustavo-1300) [reply life_expectancy/tests/test_cleaning.py]: I did some tests, with "pandas.read_table" and with "life_expectancy.cleaning.pd.read_table" it worked fine, but with "life_expectancy.cleaning.pandas.read_table" failed, so I changed to your suggestion

**Pull request merged**